std/haxe/Callback.hx

```diff
@@ -0,0 +1,62 @@
+package haxe;
+
+typedef CallbackHandler<T> = (error:Null<Error>, result:T) -> Void;
```

Comment:
Technically, `result` can also be null. As mentioned before, it would be nice if null safety could somehow deal with that.

Reply:
Nullable result would generate annecessary allocations for basic types.

Reply:
As for null safety, yes, I want to make it aware of `Callback<T>` semantics. That's why `Callback` API does not allow to provide both error and result at the same time.